### PR TITLE
py-pynetdicom3: New port

### PR DESCRIPTION
**Pull request by eborisch**
#### Description

New port: py-pynetdicom3.

###### Type(s)

- [ ] bugfix
- [X] enhancement
- [ ] security fix

###### Tested on
macOS 10.13.6 17G3025
Xcode 10.1 10B61

###### Verification 
Have you

- [X] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [X] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [ ] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [X] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [X] tried a full install with `sudo port -vst install`?
- [X] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-pynetdicom3/Portfile [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+github.setup        pydicom pynetdicom3 7ebc172
+name                py-pynetdicom3
+version             1.0.0.dev0.20181211
```
Comment (pmetzger) [inline python/py-pynetdicom3/Portfile]: The version number is a bit odd. Keep in mind that sorting order when it becomes better later on might be an issue.

File: python/py-pynetdicom3/Portfile [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+python.versions     27 36 37
+platforms           darwin
+license             MIT BSD
+maintainers         {eborisch @eborisch} \
```
Comment (pmetzger) [inline python/py-pynetdicom3/Portfile]: It's a little easier to do greps for what is and isn't openmaintainer when it's on the same line.

File: python/py-pynetdicom3/Portfile (hunk 1 above)
Comment (eborisch) [reply python/py-pynetdicom3/Portfile]: Yeah, there's no official release of 1.0.0 yet, this (up through dev0) is their internal version string currently; maybe I'll just version to 1.0.0, and keep bumping the revision whenever I update the Portfile up to and including an actual release.

Comment (pmetzger) [reply python/py-pynetdicom3/Portfile]: Not sure what the right thing is, but I do believe if we do it the other way, "1.0.0.devwhatever" will end up appearing newer than "1.0.0" which won't be what we want.

Comment (eborisch) [reply python/py-pynetdicom3/Portfile]: The latest tagged version is 0.9.1; I changed the version here to be 0.9.1-pre-1.0; hopefully this is clear enough that it is a prerelease, and this value should compare as less-than for any new tag of 0.9.2 or greater. How's that sound? Revision # can be bumped until there is an 'actual' release.

**Pull request merged**